source/whats-new.txt

```diff
@@ -32,4 +40,89 @@ What's New in 1.20
    Support for MongoDB Server v3.6 is removed in this release of the
    library.
 
-- Adds support for MongoDB Server v8.0.
\ No newline at end of file
+- Adds support for MongoDB Server v8.0.
+
+.. _php-lib-version-1.18:
+
+What's New in 1.18
+------------------
+
+- Adds a new GridFS API to make it more convenient to work with files using PHP's 
+  existing filesystem functions. The :ref:`MongoDB\GridFS\Bucket::registerGlobalStreamWrapperAlias() <api-gridfs-bucket-register-global>` 
+  method may be used to register a global alias for a GridFS bucket. After 
+  doing so, files within that bucket can be accessed using only a file URI 
+  (e.g. "gridfs://mybucket/hello.txt"). A demonstration of this API can be found
+  in the `gridfs_stream_wrapper.php <https://github.com/mongodb/mongo-php-library/blob/1.18.0/examples/gridfs_stream_wrapper.php>`__ 
+  example script.
+
+- Adds :ref:`addSubscriber() <api-client-add-subscriber>` and 
+  :ref:`removeSubscriber()<api-client-remove-subscriber>` methods to the 
+  ``MongoDB\Client`` class to make it easier to register monitoring classes 
+  on the underlying ``MongoDB\Driver\Manager`` object.
+
+To learn more about this release, see the `v1.18 Release Notes 
+<https://github.com/mongodb/mongo-php-library/releases/tag/1.18.0>`__ on GitHub.
+
+.. _php-lib-version-1.17:
+
+What's New in 1.17
+------------------
+
+- Introduces a new "codec" API for converting BSON to and from PHP objects. 
+  More information on this feature may be found in the 
+  :ref:`Codecs tutorial <php-codecs>`.
+
+- Adds :ref:`MongoDB\add_logger() <api-add-logger>` and 
+  :ref:`MongoDB\remove_logger() <api-remove-logger>` functions to the library. 
+  These functions allow applications to register a `PSR-3 Logger <https://www.php-fig.org/psr/psr-3/>`__
+  to receive log messages emitted by the driver. Previously, logs were only 
+  accessible via the extension's `mongodb.debug <https://www.php.net/manual/en/mongodb.configuration.php#ini.mongodb.debug>`__ 
+  INI setting.
+
+- Introduces new :ref:`MongoDB\Collection <api-mongodb-collection-class>` methods 
+  to create and manage :atlas:`Atlas Search indexes </atlas-search>`. Atlas 
```

Comment:
S: since this link takes readers out of the PHP docs, I'd [clarify where the link is going](https://www.mongodb.com/docs/meta/style-guide/style/links-and-cross-references/#make-link-text-clear). You could remove the link here and add a sentence to the end of this bullet point ("To learn more about Atlas Search indexes, see the :atlas:\`Atlas Search documentation </atlas-search>\`.")

Reply:
Applies to all inline links that go to the Atlas/server docs

Reply:
Thanks! What do you think of changes?

Reply:
looks good!